### ISSUE TITLE
[macOS] Request, allow the user to make a fully customizable toolbar

### ISSUE THREAD

**Issue by Oleg-Chashko** [transmission/transmission#3625]
Personal request, allow the user to make a fully customizable toolbar. 
1. Not everyone has extra space in the 13" display. 
2. Useless use of the UI. 
3. I can't remember which app on macOS Apple uses this approach to the UI.
4. I want to be able to use a minimalistic UI.

- **Toolbar**
<img width="644" alt="Do" src="https://user-images.githubusercontent.com/62497891/184222172-8b697cee-e34a-4256-b327-987dc98607ab.png">

---

- **Request Toolbar**

![22](https://user-images.githubusercontent.com/62497891/184222298-9890f978-8018-4a28-b25c-1991598a1307.png)

**Comment by nevack** [comment]
This is default behavior of Unified Toolbar Style on 11.0+

> 3. I can't remember which app on macOS Apple uses this approach to the UI.

<img width="1192" alt="Screenshot 2022-08-11 at 22 41 01" src="https://user-images.githubusercontent.com/8330119/184225880-dbe9fe7d-2f34-44b8-9af3-3b012bef5a37.png">
<img width="1393" alt="Screenshot 2022-08-11 at 22 41 33" src="https://user-images.githubusercontent.com/8330119/184225900-5f3a10e8-aab3-4015-b0d8-6ac518a01388.png">
<img width="1212" alt="Screenshot 2022-08-11 at 22 43 22" src="https://user-images.githubusercontent.com/8330119/184225906-a15237de-59ef-4c74-bb49-2afcc29fd5a8.png">

All we can do, is to add some subtitle (with upload/download speed, for example) to make it more useful.

**Comment by Oleg-Chashko** [comment]
According to your logic, it should say the name of the app, and the first two examples don't match that. 
Here are examples of how Apple should have done it. If you worked in it.

- **Finder**

<img width="1192" alt="Finder" src="https://user-images.githubusercontent.com/62497891/184228787-03654f7e-06ba-48c8-91e6-d3c319209ed7.png">

- **Mail**

<img width="1393" alt="Mail" src="https://user-images.githubusercontent.com/62497891/184228984-8ed9df79-cda7-4792-9c96-120c78ec00f4.png">

**Comment by Oleg-Chashko** [comment]
The third example does not correspond to correctness either. I was talking about applications in macOS, not utilities. I think you should understand the difference between these words.

<img width="1212" alt="Disk Utility" src="https://user-images.githubusercontent.com/62497891/184238008-bd5571a7-32b6-43c1-bcdd-516583033d02.png">

**Comment by nevack** [comment]
> The third example does not correspond to correctness either. I was talking about applications in macOS, not utilities. I think you should understand the difference between these words.

Sorry, but I don't see how Utility is different from Application.

**Comment by Oleg-Chashko** [comment]
> > The third example does not correspond to correctness either. I was talking about applications in macOS, not utilities. I think you should understand the difference between these words.
> 
> Sorry, but I don't see how Utility is different from Application.

Utility software is software designed to help analyze, configure, optimize or maintain a computer.

**Comment by GaryElshaw** [comment]
@Oleg-Chashko I can't remember if it was you and i or someone else, but this subject of removing the name from the toolbar has come up before and i even found the file you can edit to remove it if you like?

**Comment by Oleg-Chashko** [comment]
@GaryElshaw 
Yes, I've whined about this before, after the introduction of this useless innovation.
After that, I switched to an alternative application with a web interface. And with this project, I'm floating on inertia. 
Thank you for wanting to help. I know how to use Xcode. But it wasn't my intention to make a fork.
I wanted the official version to be done well. So not fate, fortunately for me, it does not matter.
